frontend/UserList.vue

```diff
@@ -172,6 +196,63 @@ export default {
         limit: 40
       }
     },
+    updateUserRoleModal: function (role) {
+      if (!!this.disableConfirmButton) {
+        return;
+      }
+
+      if (!role) {
+        return;
+      }
+
+      this.disableConfirmButton = true;
+
+      if (!this.currentUser.roles || !this.currentUser.roles.length) {
+        this.currentUser.roles = [];
+      }
+
+      var existingRoleIndex = - 1;
```

Comment:
Seems like this could be simply

    const existingRole = this.currentUser.roles.includes(role);

Reply:
Yup! This was left this way because I was relying on knowing the actual index. Now that I don't do anything with it, other than knowing if one exists then I can convert it to a boolean value.